src/main/java/core/basesyntax/service/Return.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitBox;
+import java.util.Map;
+
+public class Return extends Operation {
+    public Return(OperationType type, FruitBox fruit) {
+        super(type, fruit);
+    }
+
+    @Override
+    public Map<String, FruitBox> changeQuantity(int totalAmount,
+                                                Map<String, FruitBox> storage) {
+        for (FruitBox fruitBox: storage.values()) {
+            if (fruitBox.getDate().isEqual(fruit.getDate())) {
```

Comment:
what will happen if the return date will not be present in storage?

Reply:
...nothing special. What problem do you see?